dymension/libs/kaspa/lib/relayer/src/withdraw/sweep.rs

```diff
@@ -463,11 +469,29 @@ pub async fn create_sweeping_bundle(
         }
 
         bundle.add_pskt(pskt_signer);
+
+        // Check bundle size limit
+        let bundle_bytes = bundle
+            .serialize()
+            .map_err(|e| eyre!("Serialize bundle to check size: {}", e))?;
+        let bundle_size = bundle_bytes.len();
+
+        if bundle_size >= max_sweep_bundle_bytes {
+            info!(
+                bundle_size_bytes = bundle_size,
+                max_bundle_bytes = max_sweep_bundle_bytes,
+                pskts_count = bundle.0.len(),
+                "kaspa relayer sweeping: reached max bundle size limit, stopping"
+            );
+            break;
+        }
```

Comment:
semantically it's not ideal because it will generally make bundles ABOVE max, which is a non-standard definition of max

but nevermind